File: cpp/61A.cpp
// 61 A. Ultra-Fast Mathematician

#include <iostream>
using namespace std;

int main() {
    string a,b;
    cin >> a >> b;
    string result = "";

    for(int i=0; i< a.size(); ++i) {
        if (a[i] == b[i]) result += '0';
        else result += '1';
    }

    cout << result;
    return 0;
}

File: java/61A.java
// 61 A. Ultra-Fast Mathematician

import java.util.*;

class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String a = sc.next();
        String b = sc.next();
        StringBuilder result = new StringBuilder();

        for(int i=0; i< a.length(); i++){
            if (a.charAt(i) == b.charAt(i)){
                result.append('0');
            } else {
                result.append('1');
            }
        }

        System.out.println(result.toString());
    }
}
